### ISSUE TITLE
QSelect in IE 11 wrong display long text

### ISSUE THREAD

**Issue by alexlisitca** [quasarframework/quasar#2746]
### Software version

Operating System                Windows_NT(10.0.17134) - win32/x64
NodeJs                          8.10.0
Global packages
  NPM                           5.6.0
  yarn                          1.7.0
  quasar-cli                    0.17.20
  vue-cli                       2.9.3
  cordova                       Not installed
Browsers: IE 11.1.17134.0

Any other software related to your bug:

### What did you get as the error?
In IE 11 when you choose something data with long label text go out from input frame

### What were you expecting?
Correct visibility as I see in the other browsers

### What steps did you take, to get the error?
in template: 
 ` <q-page class="justify-center items-center">
    <div class="row">
      <div class="col-3" style="border: 1px solid black">
         <q-field>
                <q-select clearable filter filter-placeholder="Поиск" :autofocus-filter="true" float-label="Причина переоформления *"  v-model="reasonId" :options="list" />                
              </q-field>
      </div>
      <div class="col-9" style="border: 1px solid black">
        <div class="row" >
          <div class="column col-12 col-sm-10">
             <h1>Getett</h1>
            </div>
          </div>
      </div>
    </div>
  </q-page>`

in script for example: 
```
reasonId: null,
list: [ { 
 label: "gueig uegheiur gheru gher ghierhi ghhre higrhiei gergueig uegheiur gheru gher ghierhi ghhre higrhiei gergueig uegheiur gheru gher ghierhi ghhre higrhiei gergueig uegheiur gheru gher ghierhi ghhre higrhiei gergueig uegheiur gheru gher ghierhi ghhre higrhiei gergueig uegheiur gheru gher ghierhi ghhre higrhiei ger",
 value: "gueig uegheiur gheru gher ghierhi ghhre higrhiei gergueig uegheiur gheru gher ghierhi ghhre higrhiei gergueig uegheiur gheru gher ghierhi ghhre higrhiei gergueig uegheiur gheru gher ghierhi ghhre higrhiei gergueig uegheiur gheru gher ghierhi ghhre higrhiei gergueig uegheiur gheru gher ghierhi ghhre higrhiei ger"
}]
```

And similar issue we can see on your showcase if change text by devTools
![22](https://user-images.githubusercontent.com/44873803/48263956-ea990b00-e438-11e8-8bbf-f5a10f6d38ce.png)

**Comment by edgar-koster** [comment]
Sorry to bump this. But I had this too, and it causes the 'arrow' to move. My issue was actual the label or content is longer.

I added this to my app.style. May it will help with the solution. I change flex-basis to 1px instead of auto.
```
// IE
@media (-ms-high-contrast:none),screen and (-ms-high-contrast:active){.q-if>.q-if-inner{flex-basis:1px;display:inline-flex}}
```

**Comment by alexlisitca** [comment]
Oh, Thanks! Yeah, this real help me too. But now i have similar problem in Firefox 63.0.1 :)

**Comment by edgar-koster** [comment]
hmm indeed, we hadn't seen that yet. Later today I'll now something for I need to have this working today. I already see in firefox the div after the q-if-inner needs the width set to be okay, but I need some further investigation for cross browser solution

**Comment by edgar-koster** [comment]
I haven't validated just yet (I'm first checking a different issue), but if I quickly edit the css in the browsers a css line like this may work cross browser (keep the IE line above though)

`.q-if-inner > div { width: 100% }`

This means the first div after an element with class q-if-inner

Again, not checked and haven't looked where q-if-inner will be more in the display for side effects

**Comment by rstoenescu** [comment]
This has been already fixed for 1.0. Closing.
